src/main/java/org/terasology/navgraph/WalkableBlock.java

```diff
@@ -18,7 +18,8 @@
 import org.terasology.math.geom.Vector3i;
 
 /**
- * @author synopia
+ * @author synopia Represents a Block through which characters can walk through. They are blocks which have atleast 2
+ *         penetrable blocks above them and are themselves not penetrable
```

Comment:
Not sure why this is indented

Reply:
That happens as a part of automatic line formatting. I think I'll let it stay